src/util.js

```diff
@@ -13,6 +13,11 @@ export const podStates = [_("Created"), _("Running"), _("Stopped"), _("Paused"),
 
 export const fallbackRegistries = ["docker.io", "quay.io"];
 
+export function debug(system, ...args) {
+    if (window.debugging === "all" || window.debugging?.includes("podman"))
+        console.debug("podman", system ? "system" : "user", ...args);
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1327-20230703-141719-74748efe-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)